README.md

```diff
@@ -1,32 +1,36 @@
-# @guidesmiths/license-checker
+<div style="display:flex; align-items:center; justify-content:center">
+  <img alt="logo" src="./assets/banner-with-border.svg" width="100%" />
+</div>
+
+<br />
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-License checker for GuideSmiths projects
+🕵️ Audit your NPM dependencies and reject any forbidden license.
 
-## Roadmap:
-https://github.com/guidesmiths/license-checker/issues/29
+Check our [wiki](https://github.com/guidesmiths/license-checker/wiki)!
 
 ## Description
 
 This package allows you to do a quick audit on your NPM dependencies by adding it in your hooks.
 
-You can optionally add options to exclude generating the report or avoid generating the error report in case a forbidden license is found (see more details [here](#options))
+You can optionally add options to exclude generating the report or avoid generating the error report in case a forbidden license is found (see more details [here](#options)).
+
 ## How to use it in your project
 
 - Install the package
 
-```
-npm install @guidesmiths/license-checker
-```
+  ```
+  npm install @guidesmiths/license-checker
+  ```
 
 - Add a script to run the package
 
-```
-npx @guidesmiths/license-checker --failOn license1,license2
-```
+  ```
+  npx @guidesmiths/license-checker --failOn license1,license2
```

Comment:
con la version 2.0 hay que acordarse supongo que cambiar esto en el readme no ?

Reply:
CIERTO!!! Lo cambio

Reply:
Oh, I've just remembered that this has already been updated in the corresponding PR made by @neodmy